src/components/todoList/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+import '../../styles/todo-list.css';
+import '../../styles/index.css';
+import { FilterStatus } from '../../types/FilterStatus';
+
+type Props = {
+  todos: Todo [];
+  onEditTodo: (newTodo:Todo) => void;
+  onMarkCompleteOneTodo: (todoId: number) => void;
+  onTodoDelete: (todoId: number) => void;
+  statusFilter: string;
+};
+
+export const TodoList: React.FC<Props> = (
+  {
```

Comment:
A little bit confusing, component TodoList, file index.tsx. 
I think you forgot to rename something.

Reply:
i had some problems with deploy with usual approach, but now at least it works)